pkg/client/client.go

```diff
@@ -6,15 +6,48 @@ import (
 
 	azaci "github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2021-10-01/containerinstance"
 	"github.com/Azure/go-autorest/autorest"
+	"k8s.io/client-go/util/retry"
 )
 
 const (
 	DefaultUserAgent = "virtual-kubelet/azure-arm-aci"
+	APIVersion       = "2021-10-01"
 )
 
 type ContainerGroupPropertiesWrapper struct {
-	azaci.ContainerGroupProperties
-	Extensions []*Extension `json:"client,omitempty"`
+	// ProvisioningState - READ-ONLY; The provisioning state of the container group. This only appears in the response.
```

Comment:
The API needs to check on properties and gives errors. Had to revert it back to copy all proprties.